_events/2023/2023-06-community-call.md

```diff
@@ -0,0 +1,21 @@
+---
+title: June 2023 Community Call
+expires: 2023-06-10
+event_date: "June 9, 2023"
+layout: event
+duration: 60
+repeated: false
+category: community-call
+time:
+    - - start: 2023-06-09T18:00:00Z
+        end: 2023-05-11T19:00:00Z
+---
+
+The next community call will be about **Promoting Good Software Development Practices** on **Friday, June 9, 2pm ET/1pm CT/12pm MT/11am PT**.
```

Comment:
There is a whole discussion in the history of PRs about capitalizing or not capitalizing AM/PM; I don't care either way, but someone else might have a strong preference.

Reply:
I just copied the previous community call event entry, so if the official stance has changed since then, let me know and I'll update this as well.